### ISSUE TITLE
Let's get testing!

### ISSUE THREAD

**Issue by gabek** [gabek/owncast#33]
I'd like to get some different people, from the broadcasting side pointing at an instance with whatever client software they use, and see what happens.  Just as one-offs to see how things run in different stream examples, see what crazy stuff they try to do with the chat, see what kind of performance issues we run into, etc.  Just having @graywolf336 try it with some different software and environments immediately surfaced issues, so it would be nice to continue this.

What are some major things lacking before we feel comfortable testing it out with people?  Not that we'd have many people, but a few friends of mine are willing to stream to it and send a link out for one-off streams.  Plus there's always @gingervitis streaming Animal Crossing :P

My plan is to use that $10/mo machine I grabbed from Linode to host three bitrates (passthrough, medium, low) and see what happens with whatever people throw at it.

**Comment by graywolf336** [comment]
I’ll try it this weekend with an incoming stream from restream.

**Comment by gabek** [comment]
Did a simple test stream last night and it seemed to go well!  Aside from the audio, but that was my fault for having bad audio.  Next time I want to point to my other server to test out multiple bit rates, though.  Last night just had passthrough audio and video to keep it simple.  A couple people jumped in the chat, so I was glad to see that working for people for the first time.

This person https://mastodon.art/@controlfreak/104419948557737008 was interested in what we're up to and wanted to test the server, so I figure this week I'll tie up any of the "let's make sure this is in before we give it to anybody else" type things and then we can cut a 0.0.1 release to try out.  It'll be nice to have somebody download a binary and just fire it up and see how that works for them.

**Comment by graywolf336** [comment]
We can look into using a packaging system such as packr2 (https://github.com/gobuffalo/packr) to include all of the built production items into a single binary instead of distributing a zip file. Any thoughts on that?

**Comment by gabek** [comment]
I did actually think about that at first, as it would be super convenient.  But then I thought how nice it is for people to be able to edit the HTML files, swap out images, or edit the javascript right on disk in plain text.  And since an on-disk webroot is required anyway, for serving the video, having the other actual web files there is fine.  Though I'm still interested in packaging things up, as it's super handy, but maybe isn't the best fit here.

**Comment by controlfreakstudio** [comment]
I'm this person. replying as requested

happy to do a server setup out on hetzner cloud and/or also in my studio
happy to test streaming as well on multiple builds of obs in multiple distros as well as maybe restream?

**Comment by gabek** [comment]
Hey @controlfreakstudio, thanks for joining the conversation, and thanks for volunteering!

@graywolf336 in testing today just discovered an incompatibility with Restream, so I'm currently looking into resolving that.  But pointing OBS directly to Owncast should be fine.

I figured after I look at this Restream issue we can get our first release out this week for you to pull down and run.

On a side note, I don't know why I hadn't looked at your Bandcamp page until just now.  I added a couple of your releases to my cart!
